mlflow/pyfunc/__init__.py

```diff
@@ -918,7 +918,7 @@ def _get_or_create_env_root_dir(should_use_nfs):
 _MLFLOW_SERVER_OUTPUT_TAIL_LINES_TO_KEEP = 200
 
 
-def spark_udf(spark, model_uri, result_type="double", env_manager="local"):
+def spark_udf(spark, model_uri, result_type="double", env_manager=_EnvManager.VIRTUALENV):
```

Comment:
Do we need this change?

Reply:
If this is necessary, I'll create another PR.

Reply:
@harupy I'd like to include this change in this PR; we should enable env restoration by default in MLflow 2.0.

Reply:
Ok, I'll update.

Reply:
updated